src/views/v2/events-bar/form/keyword.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * View: Events Bar Form Keyword Input
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/events-bar/form/keyword.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+?>
+<div class="tribe-common-form-control-text">
+	<label for="keyword"><?php esc_html_e( 'Keyword', 'the-events-calendar' ); ?></label>
+	<input
+		type="text"
+		id="keyword"
+		name="keyword"
```

Comment:
Let's move towards using `tribe-views[search]`?

Reply:
Let's talk about this, as we never really set on how we're passing information to the BE via inputs (what format BE needs, what format works for FE, etc.)